File: Python/1015.py
import math

ponto1_x, ponto1_y = input().split(" ") # pega dois valores na mesma linha e atribui cada valor a cada variável
ponto1_x = float(ponto1_x)
ponto1_y = float(ponto1_y)

ponto2_x, ponto2_y = input().split(" ")
ponto2_x = float(ponto2_x)
ponto2_y = float(ponto2_y)

dist = ( pow(ponto2_x - ponto1_x, 2) + pow(ponto2_y - ponto1_y, 2) ) ** 0.5 # não usamos 'sqrt()', pois esta retorna um número complexo

print("%.4f" % (dist))

File: C/1015.c
#include <stdio.h>
#include <math.h>

int main(void)
{
    float x1, x2, y1, y2, pot1, pot2, raiz;
    scanf("%f %f", &x1, &y1);
    scanf("%f %f", &x2, &y2);

    pot1 = pow((x2 - x1), 2);
    pot2 = pow((y2 - y1), 2);
    raiz = sqrt(pot1 + pot2);

    printf("%.4f\n", raiz);

    return 0;
}